### PR TITLE
Translation into Brazilian Portuguese and corrections to the documeation in English.

### PR DESCRIPTION
**Pull request by luizcmarin**
Note the following:
- created folders according to the documentation standard
- added the logo at the beginning of all files *
- several link corrections (en and pt)
- various concept corrections
- see preview in https://github.com/luizcmarin/view/tree/master/docs/guides
- ...

* done to evaluate visual impact. If accepted, I will make corrections to all my forks.

### CODE REVIEW

File: docs/guides/en/basic-functionality.md [hunk 1]
```diff
@@ -76,7 +84,7 @@ Instead of an absolute file path, it accepts a name of a view in one of the foll
   the base path that was passed to `Yiisoft\View\View` constructor. For example, `/blog/posts`
   will be resolved into `/path/to/views/blog/posts.php`.
 - A name of a view without the starting slash (such as `blog/posts`). The corresponding view file will be looked for
-  in the context (instance of `Yiisoft\View\ViewContextInterface`) set via `$view->withContext()`. If the
+  in the context instance of `Yiisoft\View\ViewContextInterface` set via `$view->withContext()`. If the
```
Comment (vjik) [inline docs/guides/en/basic-functionality.md:87]: It is OK with parenthesis

File: docs/guides/en/basic-functionality.md [hunk 2]
```diff
@@ -166,8 +174,6 @@ $view->setTheme($theme);
 In a view, you can access the theme using the `getTheme()` method and manage assets as follows:
 
 ```php
-<?php
-
```
Comment (vjik) [inline docs/guides/en/basic-functionality.md:170]: `declare(strict_types=1);` can be removed also

File: docs/guides/en/use-in-web-environment.md [hunk 3]
```diff
@@ -69,11 +77,11 @@ To have scripts and tags registered and rendered correctly, special methods are
...
 
-- `POSITION_HEAD` - Corresponds to the `head()` method.
-- `POSITION_BEGIN` - Corresponds to the `beginBody()` method.
-- `POSITION_END` - Corresponds to the `endBody()` method.
-- `POSITION_READY` - Corresponds to the `endBody()` method. Located after `POSITION_END`.
-- `POSITION_LOAD` - Corresponds to the `endBody()` method. Located after `POSITION_READY`.
+- `POSITION_HEAD` - In the head section. Corresponds to the `head()` method.
+- `POSITION_BEGIN` - At the beginning of the body section. Corresponds to the `beginBody()` method.
+- `POSITION_END` - At the end of the body section. Corresponds to the `endBody()` method.
+- `POSITION_READY` - Executed when HTML document composition is ready.
```
Comment (vjik) [inline docs/guides/en/use-in-web-environment.md:83]: Keep "Located after..."

File: docs/guides/en/basic-functionality.md (hunk 2 above)
Comment (luizcmarin) [reply docs/guides/en/basic-functionality.md:170]: all occurrences in all files? Or just this one specifically?

Comment (vjik) [reply docs/guides/en/basic-functionality.md:170]: Just this one specifically.

File: docs/guides/en/use-in-web-environment.md [hunk 4]
```diff
@@ -69,11 +69,11 @@ To have scripts and tags registered and rendered correctly, special methods are
...
-- `POSITION_HEAD` - Corresponds to the `head()` method.
-- `POSITION_BEGIN` - Corresponds to the `beginBody()` method.
-- `POSITION_END` - Corresponds to the `endBody()` method.
-- `POSITION_READY` - Corresponds to the `endBody()` method. Located after `POSITION_END`.
-- `POSITION_LOAD` - Corresponds to the `endBody()` method. Located after `POSITION_READY`.
+- `POSITION_HEAD` - In the head section. Corresponds to the `head()` method.
+- `POSITION_BEGIN` - At the beginning of the body section. Corresponds to the `beginBody()` method.
+- `POSITION_END` - At the end of the body section. Corresponds to the `endBody()` method. Located after `POSITION_READY`.
+- `POSITION_READY` - Executed when HTML document composition is ready. Located after `POSITION_END`.
+- `POSITION_LOAD` - Executed when HTML page is completely loaded.
```
Comment (vjik) [inline docs/guides/en/use-in-web-environment.md:76]:
```suggestion
- `POSITION_END` - At the end of the body section. Corresponds to the `endBody()` method.
- `POSITION_READY` - Executed when HTML document composition is ready. Located after `POSITION_END`.
- `POSITION_LOAD` - Executed when HTML page is completely loaded. Located after `POSITION_READY`.
```

**Review by vjik — APPROVED** [review]

File: docs/guides/en/basic-functionality.md [hunk 5]
```diff
@@ -3,7 +3,7 @@
 The package provides a `Yiisoft\View\View` class with basic functionality for managing views, and
 a `Yiisoft\View\WebView` class with advanced functionality for use in a web environment. This guide applies to both
 classes, but examples will be provided using the `Yiisoft\View\View`. For advanced examples with
-`Yiisoft\View\WebView` functionality, see the "[Use in the web environment](use-in-web-environment.md)" guide.
+`Yiisoft\View\WebView` functionality, see the [Use in the web environment](use-in-web-environment.md) guide.
```
Comment (samdark) [inline docs/guides/en/basic-functionality.md:6]: It was alright.

File: docs/guides/en/use-in-web-environment.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # Use in the web environment
 
 This guide describes extra functionality of the `Yiisoft\View\WebView` class intended for use in a web environment.
-Please read the "[Basic Functionality](basic-functionality.md)" guide first.
+Please read the [Basic Functionality](basic-functionality.md) guide first.
```
Comment (samdark) [inline docs/guides/en/use-in-web-environment.md:4]: It was alright.

**Review by vjik — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
:+1

**Pull request merged**